src/canvas/CurveFitter.tsx

```diff
@@ -0,0 +1,64 @@
+import { Point } from './Point';
+import { Curve } from './Curve';
+import { BoundingBox } from './BoundingBox';
+
+export class CurveFitter {
+
+    static Fit = (points: Point[]): Curve => {
+        // If there are two points or less, return void becuase this cannnot be fit to a curve.
+        if (points.length <= 2) {
+            throw new Error("not enough points");
+        }
+        
+        // Get the bounds of the drawing.
+        const boundingBox = new BoundingBox(points);
+        const expandedBox = boundingBox.expand();
```

Comment:
I chose to make a separate new bounding box, because that way, the original one isn't destroyed. I was thinking there might be scenarios where you would want both the original and the expanded bounding box. However, I think it might be over engineered, since we don't currently have any scenarios like this, so I will change expand to just change the current bounding box.

Reply:
I like it now!